tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -52,6 +52,31 @@ module.exports = {
       return this.waitForElementPresent('@messageCloseIcon')
         .click('@messageCloseIcon')
         .waitForElementNotPresent('@messageCloseIcon')
+    },
+    getNotifications: async function () {
+      const notifications = []
+      await this
+        .click('@notificationBell')
+        .waitForElementVisible('@notificationElement')
+      await this.api.elements('@notificationElement', result => {
+        for (const element of result.value) {
+          this.api.elementIdText(element.ELEMENT, text => {
+            notifications.push(text.value)
+          })
+        }
+      })
+      return notifications
+    },
+    /**
+     * @param {string} expectedNotifications
+     *Checks if the notifications consists of all the expected notifications
+     */
+    assertNotificationIsPresent: async function (expectedNotifications) {
+      const notifications = await this.getNotifications()
+      for (const element of expectedNotifications) {
+        const isPresent = notifications.includes(element.title)
+        this.assert.ok(isPresent === true)
```

Comment:
This check for `true` is redundant, as `Array.prototype.includes` always returns boolean. 
```suggestion
        this.assert.ok(isPresent)
```